Calculator/CalculatorTests/CalculatorTests.swift

```diff
@@ -0,0 +1,182 @@
+//  CalculatorTests.swift
+//  CalculatorTests
+//  Created by LIMGAUI on 2022/03/15
+
+import XCTest
+@testable import Calculator
+
+final class CalculatorTests: XCTestCase {
+    var sut: LinkedList<Int>!
+    var queue: CalculateItemQueue<Int>!
+    var stringQueue: CalculateItemQueue<String>!
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        sut = LinkedList<Int>()
+        queue = CalculateItemQueue<Int>()
+        stringQueue = CalculateItemQueue<String>()
+    }
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+        sut = nil
+        queue = nil
+        stringQueue = nil
+    }
+    
+    func test_nodeInitializer_isSuccess() {
+        let nodes = Node<Int>(data: nil, next: nil)
+        
+        let result = LinkedList<Int>()
+        
+        XCTAssertEqual(result.head?.data, nodes.data)
+    }
+    
+    func test_LinkedList인스턴스가nil일때_count0이나오는지() {
+        sut.head = nil
+        
+        let result = sut.count
+        
+        XCTAssertEqual(result, 0)
+    }
+    
+    func test_LinkedList인스턴스가값이있을떄_count1이나오는지() {
+        sut.head = Node(data: 1, next: nil)
+        
+        let result = sut.count
+        
+        XCTAssertEqual(result, 1)
+    }
+    
+    func test_LinkedList인스턴스가data와next까지값가지고있을때_count2가나오는지() {
+        sut.head = Node(data: 1, next: Node(data: 1, next: nil))
+        
+        let result = sut.count
+        
+        XCTAssertEqual(result, 2)
+    }
+    
+    func test_LinkedList인스턴스Node길이가10일때_count10이나오는지() {
+        sut.head = Node(data: 1, next: Node(data: 2, next: Node(data: 3, next: Node(data: 4, next: Node(data: 5, next: Node(data: 6, next: Node(data: 7, next: Node(data: 8, next: Node(data: 9, next: Node(data: 10, next: nil))))))))))
+        
+        let result = sut.count
+        
+        XCTAssertEqual(result, 10)
+    }
+    
+    func test_append메서드3번호출시_nodeCount값이_3이나오는지() {
+        sut.append(data: 1)
+        sut.append(data: 2)
+        sut.append(data: 3)
+        
+        let result = sut.count
+        
+        XCTAssertEqual(result, 3)
```

Comment:
테스트할 데이터를 아래와 같이 관리하면 어떨까요? 혹여나 테스트할 데이터가 바뀌어도 numbers만 바꾸면되니까요.

```suggestion
        let numbers: [Int] = [1, 2, 3]
        numbers.forEach(sut.append(data:))        
        let result = sut.count
        
        XCTAssertEqual(result, numbers.count)
```

Reply:
hug!! 이렇게 사용가능한지 처음알았습니다!
sut.append(data:))메서드내에 값할당을 생략해도 되는 이유가 궁금합니다.